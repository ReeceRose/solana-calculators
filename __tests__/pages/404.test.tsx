import { render } from '@testing-library/react';

import Custom404 from '../../pages/404';

test('404 page matches snapshot', () => {
  const { asFragment } = render(<Custom404 />);
  expect(asFragment()).toMatchSnapshot();
});
