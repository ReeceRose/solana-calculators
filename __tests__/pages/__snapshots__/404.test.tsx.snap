// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="grid content-center w-full h-screen text-center"
  >
    <h1
      class="text-gray-50"
    >
      404 - Page Not Found
    </h1>
  </div>
</DocumentFragment>
`;
